src/instance-id/index.ts

```diff
@@ -38,31 +38,28 @@ import { app } from '../firebase-namespace-api';
  * var otherInstanceId = admin.instanceId(otherApp);
  *```
  *
+ * This API is deprecated. Developers are advised to use the `admin.installations()`
+ * API to delete their instance IDs and other Firebase installation IDs.
```

Comment:
Is this word required/meaningful?  If not, I'd omit it.

Reply:
Which word are you referring to? (You will have to be specific since GitHub only allows comments at line-level).

Reply:
Oh sorry -- I'm thinking of "other."  Using it suggests there may be other types of FIDs.

Reply:
Fair enough. I'll remove it.